client/src/components/NavBar.js

```diff
@@ -77,50 +94,73 @@ const NavBar = ({ history }) => {
   };
 
   return (
-    <Box color="white">
+    <div>
       <AppBar className={classes.appBar} color="inherit" position="fixed">
-        <Container maxWidth="xl">
-          <Toolbar className={classes.toolBar}>
+        <Toolbar className={classes.toolBar}>
+          <Grid
+            container
+            direction="row"
+            justify="space-between"
+            alignItems="center"
+          >
             <Logo />
             {/* <Grid item>
                 <Avatar src={userData.avatar} alt="user profile pic" />
               </Grid> */}
-
-            <Box color="#000">
+            <Grid item style={{ marginRight: "5px" }}>
+              <IconButton
+                color="inherit"
+                aria-label="navbar"
+                onClick={notifsDrawerHandler}
+              >
+                <Badge badgeContent={unreadCount} color="secondary">
+                  <NotificationsIcon fontSize="medium" />
+                </Badge>
+              </IconButton>
+            </Grid>
+            <Grid item>
               <CartIcon />
+            </Grid>
+            <Grid item>
               <IconButton
                 color="inherit"
                 aria-label="menu"
-                onClick={drawerHandler}
+                onClick={navDrawerHandler}
               >
                 <DragHandleIcon fontSize="large" />
               </IconButton>
-            </Box>
-          </Toolbar>
-
-          <Drawer
-            variant="temporary"
-            anchor="right"
-            open={open}
-            onClose={() => setOpen(false)}
-          >
-            <div className={classes.drawerDiv}>
-              <List component="nav" aria-label="navigation">
-                <ListItem divider className={classes.plateIcon}>
-                  <img src={plateLogo} alt="plate icon" />
-                </ListItem>
-                <ListItem button component={Link} to="/profile" divider>
-                  <ListItemText primary="Profile" />
-                </ListItem>
-                <ListItem button divider onClick={logoutHandler}>
-                  <ListItemText primary="Log Out" />
-                </ListItem>
-              </List>
-            </div>
-          </Drawer>
-        </Container>
+            </Grid>
+          </Grid>
+        </Toolbar>
       </AppBar>
-    </Box>
+
+      <Drawer
+        variant="temporary"
+        anchor="right"
+        open={navOpen}
+        onClose={() => setNavOpen(false)}
+      >
+        <div className={classes.drawerDiv}>
+          <List component="nav" aria-label="navigation">
+            <ListItem divider className={classes.plateIcon}>
+              <img src={plateLogo} alt="plate icon" />
```

Comment:
You could possibly use an [SvgIcon](https://material-ui.com/components/icons/#svgicon) component here

Reply:
Will do, thanks!

Reply:
It looks like you cant use a .svg file with SvgIcon